tests/integration/testAuthClient.js

```diff
@@ -46,8 +50,8 @@ test(
     }
    
    t.assert(
-     client instanceof AuthClient,
-     "The client should be an instance of AuthClient"
+     client instanceof RequestHandler,
```

Comment:
General note that I saw here: Prefer to test interface behavior instead of implementation. Unless you expect the class instance type to be important to end users, I think tests like this will end up introducing maintenance burden.